public/debugger.properties

```diff
@@ -0,0 +1,367 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This is the most interesting change since the first WIP. I completely agree that updating strings.json is an issue for both CI and in general when making a new bundle.

My current solution is to add `debugger.properties` to github. Now when we make a new bundle it's copied over. This adds a small amount of risk that we override new strings added to m-c, but we should see that in the patch pretty clearly. Also, as we get closer to officially killing the old debugger it's less of a risk.

Obviously strings.json should die soon, but that seems out of scope at the moment and we have the issue for using a webpack java properties loader.